### ISSUE TITLE
Bug: Missing src for source map generation in CRA

### ISSUE THREAD

**Issue by Dacker15** [fnando/i18n#54]
## Description

When using i18n-js with Create React App, webpack is not able to find src folder and cannot generate source map and I got warning in compilation.

## How to reproduce

1. Create a project with Create React App
2. Install the package
3. Import and use the package: simply create a new i18n class:
```
import { I18n } from 'i18n-js'

const i18n = new I18n()
```

## What do you expect

No warning should be appear in console

## What happened instead

Warning about missing file is generated

## Software:

- React version: 18.2.0
- React scripts version: 5.0.1
- 18n-js version: 4.2.2

## Full backtrace

```text
Failed to parse source map from 'C:\Users\path-to-project\node_modules\i18n-js\src\I18n.ts' file: Error: ENOENT: no such file or directory, open 'C:\Users\path-to-project\node_modules\i18n-js\src\I18n.ts'

Failed to parse source map from 'C:\Users\path-to-project\node_modules\i18n-js\src\Locales.ts' file: Error: ENOENT: no such file or directory, open 'C:\Users\path-to-project\node_modules\i18n-js\src\Locales.ts'

Failed to parse source map from 'C:\Users\path-to-project\node_modules\i18n-js\src\MissingTranslation.ts' file: Error: ENOENT: no such file or directory, open 'C:\Users\path-to-project\node_modules\i18n-js\src\MissingTranslation.ts'

Failed to parse source map from 'C:\Users\path-to-project\node_modules\i18n-js\src\Pluralization.ts' file: Error: ENOENT: no such file or directory, open 'C:\Users\path-to-project\node_modules\i18n-js\src\Pluralization.ts'

Failed to parse source map from 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\camelCaseKeys.ts' file: Error: ENOENT: no such file or directory, open 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\camelCaseKeys.ts'

Failed to parse source map from 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\createTranslationOptions.ts' file: Error: ENOENT: no such file or directory, open 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\createTranslationOptions.ts'

Failed to parse source map from 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\expandRoundMode.ts' file: Error: ENOENT: no such file or directory, open 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\expandRoundMode.ts'

Failed to parse source map from 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\formatNumber.ts' file: Error: ENOENT: no such file or directory, open 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\formatNumber.ts'

Failed to parse source map from 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\getFullScope.ts' file: Error: ENOENT: no such file or directory, open 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\getFullScope.ts'

Failed to parse source map from 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\index.ts' file: Error: ENOENT: no such file or directory, open 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\index.ts'

Failed to parse source map from 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\inferType.ts' file: Error: ENOENT: no such file or directory, open 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\inferType.ts'

Failed to parse source map from 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\interpolate.ts' file: Error: ENOENT: no such file or directory, open 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\interpolate.ts'

Failed to parse source map from 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\isSet.ts' file: Error: ENOENT: no such file or directory, open 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\isSet.ts'

Failed to parse source map from 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\lookup.ts' file: Error: ENOENT: no such file or directory, open 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\lookup.ts'

Failed to parse source map from 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\numberToDelimited.ts' file: Error: ENOENT: no such file or directory, open 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\numberToDelimited.ts'

Failed to parse source map from 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\numberToHuman.ts' file: Error: ENOENT: no such file or directory, open 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\numberToHuman.ts'

Failed to parse source map from 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\numberToHumanSize.ts' file: Error: ENOENT: no such file or directory, open 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\numberToHumanSize.ts'

Failed to parse source map from 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\parseDate.ts' file: Error: ENOENT: no such file or directory, open 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\parseDate.ts'

Failed to parse source map from 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\pluralize.ts' file: Error: ENOENT: no such file or directory, open 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\pluralize.ts'

Failed to parse source map from 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\propertyFlatList.ts' file: Error: ENOENT: no such file or directory, open 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\propertyFlatList.ts'

Failed to parse source map from 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\roundNumber.ts' file: Error: ENOENT: no such file or directory, open 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\roundNumber.ts'

Failed to parse source map from 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\strftime.ts' file: Error: ENOENT: no such file or directory, open 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\strftime.ts'

Failed to parse source map from 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\timeAgoInWords.ts' file: Error: ENOENT: no such file or directory, open 'C:\Users\path-to-project\node_modules\i18n-js\src\helpers\timeAgoInWords.ts'

Failed to parse source map from 'C:\Users\path-to-project\node_modules\i18n-js\src\index.ts' file: Error: ENOENT: no such file or directory, open 'C:\Users\path-to-project\node_modules\i18n-js\src\index.ts'

Failed to parse source map from 'C:\Users\path-to-project\node_modules\i18n-js\src\typing.ts' file: Error: ENOENT: no such file or directory, open 'C:\Users\path-to-project\node_modules\i18n-js\src\typing.ts'
```

**Comment by fnando** [comment]
For some reason it's trying to load files out of the `src` directory. I added those files to the build, so can you please try v4.2.3? I just tried with CRA and it worked.

**Comment by Dacker15** [comment]
I can confirm that the problem is solved in version 4.2.3
